spec/paperclip/has_attached_file_spec.rb

```diff
@@ -49,6 +49,10 @@
     it 'does define a media_type check if told to' do
       assert_adding_attachment('avatar').sets_up_media_type_check_validation
     end
+
+    it 'registers the activerecord reflections' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.